docs/workspaces.md

```diff
@@ -135,6 +158,9 @@ spec:
 
 #### Sharing `Workspaces` with `Sidecars`
```

Comment:
NIT: reading for the first time looks like, the two notes in the same section is contradicting each other. The first note is saying with `alpha` API, the workspaces are automatically available to sidecars  and second note is about `stable` API. With `stable` API, sidecars have to opt-in. This is fine for now, no need to change anything here. But we might have to repeat ourselves and be explicit about the behavior with `stable` compared to `alpha` 😞  May be keep the `stable` API behavior as is and add changes being introduced with `alpha` in the end of the section 🤔

Reply:
Yeah you're right it was confusing in this context.  I removed mention of the alpha api from here.